### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by ricardomlc117**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by ricardomlc117 — COMMENTED** [review]
initial thoughts

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (RicardoCarvalhoo) [inline index.html:50]: my suggestion

Comment (ricardomlc117) [inline index.html]: my suggestion
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖! Good Game!</h2>
```

**Review by ricardomlc117 — COMMENTED** [review]
aproove suggsetion

**Pull request merged**